source/includes/api/facts/reverse-sync.rst

```diff
@@ -0,0 +1,3 @@
+To reverse sync, the user must have assigned the :authrole:`restore`
+role for both source and destination and the ``enableUserWriteBlocking``
+field must be set to ``sourceAndDestination``.
```

Comment:
This is a nit, but maybe make this filename a bit more specific like `reverse-sync-requirements` or something, just in case we end up adding more `reverse` facts it's easier to distinguish.